File: src/main/java/dev/xeonkryptos/xeonrobotframeworkplugin/psi/RobotLexerExtension.java
package dev.xeonkryptos.xeonrobotframeworkplugin.psi;

public class RobotLexerExtension extends RobotLexer {

    @Override
    public void reset(CharSequence buffer, int start, int end, int initialState) {
        super.reset(buffer, start, end, initialState);
        resetLexer();
    }
}

File: src/main/kotlin/dev/xeonkryptos/xeonrobotframeworkplugin/lexer/RobotLexerExtension.kt
package dev.xeonkryptos.xeonrobotframeworkplugin.lexer

import com.intellij.openapi.project.Project
import dev.xeonkryptos.xeonrobotframeworkplugin.psi.RobotLexer

class RobotLexerExtension(project: Project? = null) : RobotLexer(project) {

    override fun reset(buffer: CharSequence?, start: Int, end: Int, initialState: Int) {
        super.reset(buffer, start, end, initialState)
        resetLexer()
    }
}
